packages/client/katello-pull-provider-migrate/katello-pull-provider-migrate.spec

```diff
@@ -0,0 +1,43 @@
+Name:           katello-pull-provider-migrate
+Version:        1.0.0
+Release:        1%{?dist}
+Summary:        An RPM that migrates katello-agent users to the new pull provider
+BuildArch:      noarch
+
+License:        LGPLv2
+URL:            https://github.com/theforeman/foreman-packaging/tree/rpm/develop/packages/client/katello-pull-provider-migrate
+Source0:        %{name}
+
+Requires:       bash
+Requires:       yggdrasil
+Requires:       foreman_ygg_worker
+
+%description
+An RPM that migrates katello-agent users to the new pull provider.
+
+Relies on the existing Red Hat Subscription Manager configuration to know which Katello server it should communicate with.
+The scriptlet will fail if the client is not registered to a Katello server.
```

Comment:
I would either drop `Katello`, use `Foreman` instead. Or `to a Foreman server with the Katello plugin.` I generally want us all to get away from calling it a Katello server and think of it as a Foreman server with X functionality (in this case Katello).